tests/test_range.py

```diff
@@ -646,6 +646,52 @@ def test_in(self):
                           datetime(2017, 6, 16, 19, 38, 7),
                           datetime(2017, 6, 16, 19, 38, 8))
 
+    def test_since(self):
+        now = datetime.now()
+        self.assert_range('since last may', datetime(2017, 5, 1), now)
+        self.assert_range('since last Friday', datetime(2017, 6, 9), now)
+        self.assert_range('since last Saturday', datetime(2017, 6, 10), now)
+        self.assert_range('since last Thursday', datetime(2017, 6, 15), now)
+        self.assert_range('since last year', datetime(2016, 1, 1), now)
+        self.assert_range('since last month', datetime(2017, 5, 1), now)
+        self.assert_range('since last week', datetime(2017, 6, 5), now)
+
+        self.assert_range('since Friday', datetime(2017, 6, 9), now)
+        self.assert_range('since Saturday', datetime(2017, 6, 10), now)
+        self.assert_range('since Thursday', datetime(2017, 6, 15), now)
+
+        self.assert_range('since 2016', datetime(2016, 1, 1), now)
+        self.assert_range('since July', datetime(2016, 7, 1), now)
+        self.assert_range('since April 2016', datetime(2016, 4, 1), now)
+        self.assert_range('since April 11, 2016', datetime(2016, 4, 11), now)
+
```

Comment:
What would happen if I say "Since April 2020"? We should be sure to handle errors like this